frontend/src/metabase/admin/settings/containers/SettingsEditorApp.jsx

```diff
@@ -82,8 +83,12 @@ export default class SettingsEditorApp extends Component {
             }
 
             this.refs.layout.setSaved();
-            let val = (setting.key === "report-timezone" || setting.type === "boolean") ? setting.value : "success";
-            MetabaseAnalytics.trackEvent("General Settings", setting.display_name || setting.key, val);
+
+            MetabaseAnalytics.trackEvent(
+                "General Settings",
```

Comment:
@salsakran Yup. This method gets called for every part of the admin settings when a setting gets updated. I think the `General Settings` bit here is vestigial but I'm not sure if changing it would screw up our history so I left it alone for now.